app/questionPage.html

```diff
@@ -18,6 +18,8 @@
     <link href="css/bootstrap.min.css" type="text/css" rel="stylesheet">
     <link href="css/questionPage.css" type="text/css" rel="stylesheet">
     <link rel="stylesheet" href="css/resultPage.css">
+    <script src="js/utils.js" defer></script>
+    <script src="js/getData.js" defer></script>
```

Comment:
Yeah, at some point I wonder if we should just go over all of the script tags and add a defer keyword